content/applications/marketing/events/create_events.rst

```diff
@@ -0,0 +1,374 @@
+   modifications.
+
+Add or modify the cost of the ticket in the :guilabel:`Price` field. Then, determine a
+:guilabel:`Sales Start` and :guilabel:`Sales End` date in their respective fields. To do that, click
+into the blank field to reveal a calendar popover. From there, select the desired date and time,
+then click :guilabel:`✔️ Apply`.
+
+Then, if desired, designate a :guilabel:`Maximum` amount of that specific ticket that can be sold.
+
+The :guilabel:`Taken` column will populate with accurate data as tickets are sold.
+
+In the :guilabel:`Color` column, users have the option to add a custom color code to different
+tickets.
+
+If the user wishes to delete any tickets from the :guilabel:`Tickets` tab, click the :guilabel:`🗑️
+(trash can)` icon on the corresponding line for the ticket that should be deleted.
+
+.. tip::
+   To add an optional :guilabel:`Description` column to the :guilabel:`Tickets` tab, click the
+   :guilabel:`(optional columns drop-down)` menu, located to the far-right of the column titles.
+   Then, tick the checkbox beside :guilabel:`Description` from the resulting drop-down menu.
+
+   When added, users have the option to add brief descriptions for each event ticket created, which
+   can be used to inform attendees of any perks or amenities that may coincide with specific tickets.
+
+.. _create_events/event_form/communication:
+
+Communication tab
+-----------------
+
+In the :guilabel:`Communication` tab of an event form, users can create various communications that
+can be scheduled to be sent at specific intervals leading up to, and following, the event.
+
+.. image:: create_events/communication-tab.png
+   :align: center
+   :alt: Typical communication tab on an event form in the Odoo Events application.
+
+.. note::
+   By default, Odoo provides three separate pre-configured communications on every new event form.
+
+To add a communication in the :guilabel:`Communication` tab, click :guilabel:`Add a line`. Then,
+select the desired type of communication in the :guilabel:`Send` field. The options are:
+:guilabel:`Mail`, :guilabel:`SMS`, :guilabel:`Social Post`, or :guilabel:`WhatsApp`.
+
+.. important::
+   The :guilabel:`Social Post` option **only** appears if the *Social Marketing* application is
+   installed.
+
+   The :guilabel:`WhatsApp` option **only** appears if the *WhatsApp Integration* module is
+   installed.
+
+Then, either select a communication template from the :guilabel:`Template` drop-down menu.
+
+.. note::
+   These existing communication templates can be modified directly from this field, if necessary, by
+   selecting the chosen template, and clicking the :guilabel:`Internal link ➡️ (right arrow)` icon
+   to the right. Doing so reveals a separate page where users can edit the :guilabel:`Content`,
+   :guilabel:`Email Configuration`, and :guilabel:`Settings` of that particular communication
+   template.
+
```

Comment:
```suggestion
   These existing communication templates can be modified directly from this field, if necessary, by
   selecting the chosen template, and clicking the :guilabel:`Internal link ➡️ (right arrow)` icon
   to the right. Doing so reveals a separate page where users can edit the :guilabel:`Content`,
   :guilabel:`Email Configuration`, and :guilabel:`Settings` of that particular communication
   template.

```

Just a note WhatsApp templates cannot be edited on the fly, they require separate approval from Meta.

Reply:
Oh, good to know. Just added an admonition for that super-helpful bit of knowledge. Thanks for the heads up! 👍